src/test/java/org/broadinstitute/hellbender/utils/StringSplitSpeedUnitTest.java

```diff
@@ -19,7 +19,7 @@
             "Italiam, fato profugus, Laviniaque venit " +
             "litora, multum ille et terris iactatus et alto " +
             "vi superum saevae memorem Iunonis ob iram; " +
-            "multa quoque et bello passus, dum conderet urbem, " +
+            "multa quoque et bello passus, testConfigurationSorting conderet urbem, " +
```

Comment:
was this maybe a global search/replace change? My high school latin is pretty rusty but I don't think `testConfigurationSorting` is a word that can replace a conjunction like 'dum'. :)

Reply:
definitely better than mine, who initially named a function "dum" :cold_sweat: